modules/ossm-using-discoveryselectors-scope-service-mesh.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+// install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-discoveryselectors-scope-service-mesh_{context}"]
+= Using DiscoverySelectors to scope a Service Mesh
+:context: ossm-discoveryselectors-scope-service-mesh
+
+If you know which namespaces to include in the {SMProductShortName}, configure `discoverySelectors` during or after installation by adding the required selectors to the `meshConfig.discoverySelectors` section of the `{istio}` resource. For example, configure {istio} to discover only namespaces labeled `istio-discovery=enabled`.
+
+.Prerequisites
+* The {SMProduct} operator is installed
+* An {istio} CNI resource is created
+
+.Procedure
+. Add a label to the namespace containing the {istio} control plane, for example, the `istio-system` system namespace.
++
+[source,terminal]
+----
+$ oc label namespace istio-system istio-discovery=enabled
+----
+
+. Modify the `{istio}` control plane resource to include a `discoverySelectors` section with the same label.
++
+[source,yaml]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            istio-discovery: enabled
+----
+
+. Apply the Istio CR:
++
+[source,terminal]
+----
+$ oc apply -f istio.yaml
+----
+
+. Ensure that all namespaces that will contain workloads that are to be part of the {SMProductShortName} have both the `discoverySelector` label and, if desired, the appropriate {istio} injection label.
+
```

Comment:
Should the following note be nested under step 4?

Reply:
I don't think so. Th note in the end is a note for all the whole procedure. Also the last step is connected to the **Next Steps** mentioned after this module.